test/multi_search_test.exs

```diff
@@ -0,0 +1,76 @@
+
+  setup_all do
+    conn = %ExTypesense.Connection{
+      host: "localhost",
+      api_key: "xyz",
+      port: 8108,
+      scheme: "http"
+    }
```

Comment:
you can remove this connection struct, as I've already added the connection string in `test/test_helper.exs`